app/gui/src/model/execution_context.rs

```diff
@@ -216,6 +217,78 @@ pub struct LocalCall {
 
 
 
+// ==============================
+// === QualifiedMethodPointer ===
+// ==============================
+
+/// A method pointer containing the qualified module and type names.
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Is there a reason not to also have `Eq`? (This will soon be a Clippy lint).

Reply:
Didn't know that it's necessary. I'll add one